### PR TITLE
use try catch

### PR DESCRIPTION
**Pull request by linefile**
Before :
before
Fetching state not reset when there is an error (e.g. SyntaxError: Unexpected end of JSON input)

After :
after

### CODE REVIEW

File: frontend/src/components/Editor/editor.tsx [hunk 1]
```diff
@@ -48,40 +48,51 @@ export default function Editor(): JSXElement {
...
       const imageBlob = await imageResponse.blob();
       setImage(await blobToBase64(imageBlob));
-      setError('');
-    } else {
-      const errorMsg = await imageResponse.json();
-      setError(errorMsg.msg.toString());
+    } catch (err) {
+      if (err instanceof Error) {
+        // eslint-disable-next-line no-console
+        console.log(err);
```
Comment (elianiva) [inline frontend/src/components/Editor/editor.tsx:88]: `console.error` aja

**Review by elianiva — APPROVED** [review]

**Comment by elianiva** [comment]
image

**Comment by elianiva** [comment]
thanks

**Comment by linefile** [comment]
sorry ada yg ketinggalan, harusnya clear error dulu kalo responsenya success

**Comment by elianiva** [comment]
yaudah gpp, bikin PR baru aja kalo mau fix

**Pull request merged**